docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -0,0 +1,14 @@
+gluon-config-mode-geo-location
+==============================
+
+This package enables the user to set latitude, longitude and altitude of their
+node within config mode. As the usage of the altitude is not well defined the
+corresponding field can be disabled.
+
+site.conf
+---------
+
+config_mode.geo_location.show_altitude
+    - ``true`` enables the altitude field
+    - ``false`` disables the altitude field
+    - if not set, the altitude field is enabled, too
```

Comment:
Just write "defaults to true" and mention that this setting is optional / not required.